pom.xml

```diff
@@ -47,17 +47,9 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.mortbay.jetty.alpn</groupId>
-            <artifactId>alpn-boot</artifactId>
-            <version>${jetty.alpn.version}</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
             <groupId>org.eclipse.jetty.alpn</groupId>
             <artifactId>alpn-api</artifactId>
             <version>1.1.2.v20150522</version>
-            <!-- Provided by alpn-boot; see
-            http://www.eclipse.org/jetty/documentation/9.2.8.v20150217/alpn-chapter.html#alpn-understanding -->
             <scope>provided</scope>
```

Comment:
Forgive me if I'm being dense; I was just trying to write an update for the README that explains *why* it's not needed, but I'm not quite finding what I think you're referring to on the `jetty-alpn-agent` project page. I hate to be needy, but can you be a little more specific about where I should be looking?

Thanks!